src/fmu/dataio/models/meta/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .model import Root, dump
```

Comment:
May cause confusion when the word "model" hits the namespace in a context where this word has lots of meanings. Possible to use "datamodels"?

Reply:
"datastructure" or "datastructures"? Ref discussion